### PR TITLE
Redesign confirm forms

### PR DESCRIPTION
**Pull request by NicolasDorier**
From now, all confirm form would have this design:

image

ACK?

### CODE REVIEW

**Comment by NicolasDorier** [comment]
Ping @pavlenex

**Comment by pavlenex** [comment]
Concept ACK, will test this in a few hours.

**Comment by pavlenex** [comment]
Design wise I like it. One thing that may be useful specifically when deleting a new store is some sort of back button. This way user has no choice where to click except `Delete` which looks like CTA. Of course they can go back in the browser, but still not a perfect UX.

**Comment by britttttk** [comment]
ACK

File: BTCPayServer/Views/Shared/Confirm.cshtml [hunk 1]
```diff
@@ -1,26 +1,55 @@
...
             </div>
-        </div>
-        @if (!String.IsNullOrEmpty(Model.Action))
-        {
-        <div class="row">
-            <div class="col-lg-12 text-center">
-                <form method="post">
-                    <button id="continue" type="submit" class="btn btn-secondary @Model.ButtonClass" title="Continue">@Model.Action</button>
-                </form>
+            <div class="modal-body">
```
Comment (britttttk) [inline BTCPayServer/Views/Shared/Confirm.cshtml]: `text-center` here looks better to me

Example Without:
image

**Comment by NicolasDorier** [comment]
Added a `Go back` button and using text-center.

**Pull request merged**